src/platform/site-wide/sass/modules/_m-footer.scss

```diff
@@ -154,8 +154,8 @@ body.va-pad-promo-banner .footer-inner {
   cursor: pointer;
   display: block;
   margin: 0;
-  padding: 1.6rem 0.8rem;
-  font-size: 1.6rem;
+  padding: 16px 8px;
```

Comment:
Should this (and the line below) be in `rem`?

Reply:
@randimays this should be in `px` because it targets the footer in va.gov and once the root font size changes the effective values will change too.

Reply:
I might be misunderstanding; isn't that the point of converting everything else in this PR to `rem`?

Reply:
I'll let @it-harrison chime in, but I think we found some irregularities with how the injected header and footer for teamsites interacts with the `scale-rem` helper we've been using. Because of this, in several places for the header and footer, we've elected to convert rem values to their pixel equivalents. 

Here's the original issue for that - https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2730

And here's the PR we have into formation, which will eventually make it's way into this branch here in vets-website - https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/1036 (it includes some unrelated css for forms because we're trying to really limit the number of formation releases we have to do :sweat_smile: )